### PR TITLE
[W5.4][F11-A1]Phua Han Siang

### PR DESCRIPTION
**Pull request by hansiang93**

### CODE REVIEW

**Comment by hansiang93** [comment]
@karrui @sofarsophie ready for review :)

File: src/seedu/addressbook/commands/SortCommand.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Displays all persons in the address book as a list with index numbers in a sorted manner.\n"
+            + "Example: " + COMMAND_WORD;
+
+
```
Comment (PierceAndy) [inline src/seedu/addressbook/commands/SortCommand.java]: Extra line

File: src/seedu/addressbook/data/AddressBook.java [hunk 2]
```diff
@@ -128,4 +128,16 @@ public boolean equals(Object other) {
                         && this.allPersons.equals(((AddressBook) other).allPersons)
                         && this.allTags.equals(((AddressBook) other).allTags));
     }
+    /**
```
Comment (PierceAndy) [inline src/seedu/addressbook/data/AddressBook.java]: Missing new line after closing brace and before header comment.

File: src/seedu/addressbook/data/person/UniquePersonList.java [hunk 3]
```diff
@@ -1,11 +1,6 @@
 package seedu.addressbook.data.person;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.List;
+import java.util.*;
```
Comment (PierceAndy) [inline src/seedu/addressbook/data/person/UniquePersonList.java]: Import statements should be listed explicitly.
See point 3 here: https://github.com/oss-generic/process/blob/master/codingStandards/CodingStandard-Java.adoc#statements

File: src/seedu/addressbook/data/person/UniquePersonList.java [hunk 4]
```diff
@@ -21,6 +16,24 @@
...
+     */
+    public void sort() {
+        Collections.sort(internalList, new Comparator<Person>(){
+            @Override
+            public int compare(Person o1, Person o2){
+                return o1.compareTo(o2);
+            }
+        });
+    }
+    /**
```
Comment (PierceAndy) [inline src/seedu/addressbook/data/person/UniquePersonList.java]: Missing new line after closing brace and before header comment.

File: src/seedu/addressbook/parser/Parser.java [hunk 5]
```diff
@@ -11,17 +11,7 @@
...
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```
Comment (PierceAndy) [inline src/seedu/addressbook/parser/Parser.java]: Import statements should be listed explicitly.
See point 3 here: https://github.com/oss-generic/process/blob/master/codingStandards/CodingStandard-Java.adoc#statements

File: test/java/seedu/addressbook/commands/SortCommandTest.java [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+import seedu.addressbook.data.person.Address;
+import seedu.addressbook.data.person.Email;
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.Phone;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.UniquePersonList.PersonNotFoundException;
+import seedu.addressbook.data.tag.UniqueTagList;
+import seedu.addressbook.ui.TextUi;
+import seedu.addressbook.util.TestUtil;
```
Comment (PierceAndy) [inline test/java/seedu/addressbook/commands/SortCommandTest.java]: There are several unused import statements here.

**Comment by PierceAndy** [comment]
@hansiang93 Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**